frontend/src/pages/Login.tsx

```diff
@@ -23,13 +23,12 @@ export const Login = () => {
       const data = await response.json(); //Parse JSON response
       localStorage.setItem("token", data.token); //Save the token in localStorage
       navigate("/home");
-      // if (username === "user" && password === "password") {
-      // navigate("/home");
-      //} else {
-      //  alert("Invalid credentials");
-      //}
     } catch (error) {
       alert("Invalid credentials");
```

Comment:
```suggestion
      console.error("Loggin error:", error);
      alert("Invalid credentials");
```

Reply:
It is a good practice to console the error, so in case it fails, developers can understand easily what is wrong getting feedback on the console.